go/vt/vtgate/planbuilder/operators/dml_planning.go

```diff
@@ -41,6 +39,35 @@ type TargetTable struct {
 	Name   sqlparser.TableName
 }
 
+// dmlOp stores intermediary value for Update/Delete Operator with the vindexes.Table for ordering.
+type dmlOp struct {
+	op   Operator
+	vTbl *vindexes.Table
+	cols []*sqlparser.ColName
```

Comment:
what are we storing in `cols`?

Reply:
They store the select columns to get from the input.